src/test/java/com/google/cloud/genomics/dataflow/utils/VariantUtilsTest.java

```diff
@@ -34,7 +34,16 @@
   public void testIsVariant() {
     assertTrue(VariantUtils.isVariant(DataUtils.makeVariant("chr7", 200000, 200001, "A",
```

Comment:
Thanks for taking a look.

@jean-philippe-martin Here is the test for SNPs.  isVariant returns true.  I will re-write isVariant to be more readable.

@pgrosu the purpose of that method is to differentiate between variants and non-variant segments (a.k.a. non-variant block records).  For Complete Genomics data and gVCFs such as Platinum Genomes, we wind up with zero alternates (the missing value alternate `.` in the VCF spec gets converted to null).  That reminds me, I should add some logic for `<NON_REF>` which is emitted by https://www.broadinstitute.org/gatk/guide/article?id=4017